tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Remote Staking via XCM
+description: In this guide, we'll be leveraging remote execution to remotely stake GLMR on Moonbeam using a series of XCM instructions.
+---
+
+# Remote Staking via XCM
+
+![Banner Image](/images/tutorials/remote-staking-via-xcm/remote-staking-via-xcm-banner.png)
+
+## Introduction {: #introduction } 
+In this tutorial, we’ll stake DEV tokens remotely by sending XCM instructions from an account on the Moonbase Relay Chain (equivalent to the Polkadot Relay Chain). This tutorial assumes a basic familiarity with [XCM](/builders/xcm/overview/){target=_blank} and [Remote Execution via XCM](builders/xcm/xcm-transactor/){target=_blank}. You don’t have to be an expert on these topics but you may find it helpful to have some XCM knowledge as background. 
```

Comment:
I guess we can use 2nd person because it is a blogpost style article? Or should we keep it like other tutorials? (cc @eshaben )